### ISSUE TITLE
Script/Ulduar: mob waves during Thorim seem to be wrong for 10-man

### ISSUE THREAD

**Issue by Wyrserth** [TrinityCore/TrinityCore#23626]
**Description:**

Right now the number of adds spawned in the arena (between 2 and 4) and the timer at which they are spawned (10 seconds) are the same for both 10-man and 25-man versions of the Thorim encounter.

I haven't been able to gather much meaningful information about this, but from videos from 2009 it does look like that on 10-man only 1-3 adds spawn. The timer seems to be ~14 seconds.

[The only good video I could find about 10-man Thorim](https://www.youtube.com/watch?v=o4__jtI--N0).

**Branch(es):** both.

**TC rev. hash/commit:** ef3ee63d880240fcefcac81d0f0f44d854b47625

**Comment by Aokromes** [comment]
anyone have any idea on this?

**Comment by Rushor** [comment]
has this been changed on retail since wotlk?

**Comment by Shauren** [comment]
Yes, it was changed when ulduar timewalking was implemented and 10&25 modes were merged
